doc/changes.md

```diff
@@ -1,4 +1,136 @@
 # Changelog
+## Release v0.4.0 (2021/06/20)
+
+[![Digital Object Identifier for PyGMT v0.4.0](https://zenodo.org/badge/DOI/10.5281/zenodo.4978645.svg)](https://doi.org/10.5281/zenodo.4978645)
+
+### Highlights
+
+* 🎉 **Fourth minor release of PyGMT** 🎉
+* 10 new PyGMT functions and methods
+* New and improved gallery examples and tutorials
```

Comment:
Wonder if we can come up with a more catchy highlight item :slightly_smiling_face: Probably list the datetime and vector tutorials? And we probably don't need to mention that there's 10 new wrapped functions since it's the next item down.

Reply:
> Wonder if we can come up with a more catchy highlight item  Probably list the datetime and vector tutorials?

Totally agreed. Those two tutorials are great and very helpful. Perhaps we could highlight a few tutorial/gallery for each release.

> And we probably don't need to mention that there's 10 new wrapped functions since it's the next item down.

Agreed.

Reply:
The datetime and vector tutorials are definetly worth to mention them as highlights.

Reply:
Done in https://github.com/GenericMappingTools/pygmt/pull/1341/commits/3cdceb364d707c16bb4db23dc7adfd0abc164857